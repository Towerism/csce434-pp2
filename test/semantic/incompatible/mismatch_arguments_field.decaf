class Asdf {
      int asdf(int a, int b, int c) {}
}
int fdsa() {
    int a;
    int b;
    Asdf c;
    Asdf d;
    d.asdf(a, b, c);
}