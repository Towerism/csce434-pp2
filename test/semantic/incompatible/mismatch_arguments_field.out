
*** Error line 9.
    d.asdf(a, b, c);
                 ^
*** Incompatible argument 3: Asdf given, int expected

